spec/models/document_type/summary_field_spec.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
This could be fixed up to the commit where the spec file is introduced.

Reply:
Sure - fixed.